app/actions/sessions.js

```diff
@@ -3,8 +3,6 @@ import uuid from 'node-uuid';
 import {createAction} from 'redux-actions';
```

Comment:
what will happen when users who already have self-signed certificates install the upgraded connector?